tests/flows/test_helpers.py

```diff
@@ -2303,3 +2308,274 @@ def test_validate_sku_availability_empty_sku_lists(
 
     mock_next_step.assert_called_once_with(mock_mpt_client, context)
     mocked_get_prices.assert_called_once_with("PRD-123", "USD", [])
+
+
+def test_validate_government_transfer_skips_when_not_government_segment(
+    mocker,
+    order_factory,
+    mock_next_step,
+    mock_mpt_client,
+):
+    """Test that validation is skipped when market segment is not government."""
```

Comment:
Please remove the comment as it's not needed

Same comment here about AAA pattern ;)